dali/base/dautils.cpp

```diff
@@ -557,23 +554,18 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    if (isExternalFile(name, lfn, false, this->tailpos))
-    {
-        //lfn.set(name);
-        //const char *ns1 = strstr(name,"::>");
-        //if (ns1)
-            //tailpos = ns1 - name + 2;
-        //else
-        //    lfn.toLowerCase();
+
+    if (normalizeExternal(name, false))
```

Comment:
Also, it's inconsistent with normalizeName. normalizeName returns the formatted string, where as normalizeExternal sets the member 'lfn'.

They should be consistent to avoid confusion.